modules/ppcp-save-payment-methods/services.php

```diff
@@ -20,75 +20,56 @@
 		$save_payment_methods_applies = $container->get( 'save-payment-methods.helpers.save-payment-methods-applies' );
 		assert( $save_payment_methods_applies instanceof SavePaymentMethodsApplies );
 
-		return $save_payment_methods_applies->for_country_currency();
+		return $save_payment_methods_applies->for_country();
 	},
 	'save-payment-methods.helpers.save-payment-methods-applies' => static function ( ContainerInterface $container ) : SavePaymentMethodsApplies {
 		return new SavePaymentMethodsApplies(
-			$container->get( 'save-payment-methods.supported-country-currency-matrix' ),
-			$container->get( 'api.shop.currency.getter' ),
+			$container->get( 'save-payment-methods.supported-countries' ),
 			$container->get( 'api.shop.country' )
 		);
 	},
-	'save-payment-methods.supported-country-currency-matrix' => static function ( ContainerInterface $container ) : array {
-		$default_currencies = array(
-			'AUD',
-			'BRL',
-			'CAD',
-			'CHF',
-			'CZK',
-			'DKK',
-			'EUR',
-			'GBP',
-			'HUF',
-			'ILS',
-			'JPY',
-			'MXN',
-			'NOK',
-			'NZD',
-			'PHP',
-			'PLN',
-			'SEK',
-			'THB',
-			'TWD',
-			'USD',
-		);
+	'save-payment-methods.supported-countries'           => static function ( ContainerInterface $container ) : array {
+		if ( has_filter( 'woocommerce_paypal_payments_save_payment_methods_supported_country_currency_matrix' ) ) {
+			_deprecated_hook( 'woocommerce_paypal_payments_save_payment_methods_supported_country_currency_matrix', '3.0.0', 'woocommerce_paypal_payments_save_payment_methods_supported_countries', esc_attr__( 'Please use the new Hook to filer countries for saved payments in PayPal Payments.', 'woocommerce-paypal-payments' ) );
```

Comment:
There is a typo in "Please use the new Hook to **filer**", it should be **filter**.